src/aat/resources/environment-solicitor.properties

```diff
@@ -0,0 +1,2 @@
+
+oauth2.client.id= cmc_legal
```

Comment:
Is it worth adding the complexity that this causes?
There is nothing special about different oauth client ids and they cause no difference in the outcome of a login,
I would recommend removing this and the BaseCitizen and BaseSolicitor classes.

Reply:
I asked @dharmendrak about this and was recommended to use `cmc_legal` for the solicitor requests - I guess for the sake of realism.

Please confirm. (I'm hoping you do, because I can make a nicer abstraction that way!)

Reply:
its got nothing to do with realism here though, all you need is an authenticated user. 
You could use the client id muppet and get the same outcome.
IMO its unneeded complexity in the tests

Reply:
Splendid. On it